### PR TITLE
Add raw and cbor codecs to the lookup in ipfs repo

### PR DESCRIPTION
**Pull request by icidasset**
Missed a few codecs.

### CODE REVIEW

File: src/ipfs/node/repo.ts [hunk 1]
```diff
@@ -14,7 +16,11 @@ export function create(): IPFSRepo {
     codeOrName => {
       const lookup: Record<string, BlockCodec<number, unknown>> = {
         [dagPB.code]: dagPB,
-        [dagPB.name]: dagPB
+        [dagPB.name]: dagPB,
+        [dagCBOR.code]: dagCBOR,
+        [dagCBOR.name]: dagCBOR,
+        [raw.code]: raw,
+        [raw.name]: raw,
```
Comment (matheus23) [inline src/ipfs/node/repo.ts:23]: You can close this. They got in here through #389 
https://github.com/fission-codes/webnative/blob/2d0a93e4f435a51a7618ce47544fb17b391352ad/tests/helpers/in-memory-ipfs.ts#L16-L23

Comment (icidasset) [reply src/ipfs/node/repo.ts:23]: Hmm, but that's another file/repo/ipfs-instance?

Comment (matheus23) [reply src/ipfs/node/repo.ts:23]: Nevermind me!
Yeah, this change is in the `src/ipfs/node/repo.ts` file.
My changes are from the `tests/helpers/in-memory-ipfs.ts` file.
Whoops!

**Review by matheus23 — APPROVED** [review]
:shipit:

**Pull request merged**